docs/index.md

```diff
@@ -2,18 +2,36 @@
 
 ## About
 
-A new kind of BIOS firmware that will solve the problems of ownership, security
-and compatibility allowing you create secure, efficient images that are fully
-customizable to your tastes and needs.
+Dasharo is open source firmware distribution focusing on clean and simple code,
+long-term maintenance, transparent validation, privacy-respecting
+implementation, liberty for the owners, and trustworthiness for all.
 
-Dasharo is a set of productized services, Open Core, and SaaS products which
-help to provide scalable, modular, easy to combine Open Source BIOS, UEFI, and
-Firmware solutions. It offers the components that are needed to develop and
+Dasharo consist of productized services, open-core, and SaaS products which
+help to provide scalable, modular, easy to combine open-source BIOS, UEFI, and
```

Comment:
We use `open-source` here but `open source` in line 5

Reply:
You're right - fixed that.